### ISSUE TITLE
Name of the input node?

### ISSUE THREAD

**Issue by sigmoidx** [balancap/SSD-Tensorflow#248]
Hi, I was checking the pb file converted from ckpt file provided in this repository.

```
**./summarize_graph --in_graph=ssd300_frozen_graph.pb**
No inputs spotted.
No variables spotted.
Found 1 possible outputs: (name=ssd_losses/softmax/Softmax, op=Softmax) 
Found 25750961 (25.75M) const parameters, 0 (0) variable parameters, and 0 control_edges
Op types used: 99 Const, 59 Identity, 29 BiasAdd, 29 Conv2D, 23 Relu, 13 Reshape, 6 Transpose, 6 Mul, 5 MaxPool, 4 Add, 2 Pad, 2 Sub, 2 RealDiv, 2 RandomUniform, 2 Floor, 2 ExpandDims, 1 QueueDequeueV2, 1 Maximum, 1 Range, 1 FIFOQueueV2, 1 Rsqrt, 1 Softmax, 1 SpaceToBatchND, 1 Square, 1 ConcatV2, 1 Sum, 1 BatchToSpaceND
To use with tensorflow/tools/benchmark:benchmark_model try these arguments:
bazel run tensorflow/tools/benchmark:benchmark_model -- --graph=ssd300_frozen_graph.pb --show_flops **--input_layer=** --input_layer_type= --input_layer_shape= **--output_layer=ssd_losses/softmax/Softmax**
```

As you see, it supposed to display input and output nodes but it returns the output layer only. Can you tell me what the input node name is?

Thanks,

**Comment by jeffreyscholz** [comment]
That will depend on how you froze the graph -- if you included the preprocessing or not. Try `Placeholder:0` if you froze the preprocessing.

**Comment by lucky-xu-1994** [comment]
> Hi, I was checking the pb file converted from ckpt file provided in this repository.
> **./summarize_graph --in_graph=ssd300_frozen_graph.pb**
> No inputs spotted.
> No variables spotted.
> Found 1 possible outputs: (name=ssd_losses/softmax/Softmax, op=Softmax) 
> Found 25750961 (25.75M) const parameters, 0 (0) variable parameters, and 0 control_edges
> Op types used: 99 Const, 59 Identity, 29 BiasAdd, 29 Conv2D, 23 Relu, 13 Reshape, 6 Transpose, 6 Mul, 5 MaxPool, 4 Add, 2 Pad, 2 Sub, 2 RealDiv, 2 RandomUniform, 2 Floor, 2 ExpandDims, 1 QueueDequeueV2, 1 Maximum, 1 Range, 1 FIFOQueueV2, 1 Rsqrt, 1 Softmax, 1 SpaceToBatchND, 1 Square, 1 ConcatV2, 1 Sum, 1 BatchToSpaceND
> To use with tensorflow/tools/benchmark:benchmark_model try these arguments:
> bazel run tensorflow/tools/benchmark:benchmark_model -- --graph=ssd300_frozen_graph.pb --show_flops **--input_layer=** --input_layer_type= --input_layer_shape= **--output_layer=ssd_losses/softmax/Softmax**
> 
> As you see, it supposed to display input and output nodes but it returns the output layer only. Can you tell me what the input node name is?
> Thanks,

have you solved this problem? I have the same problem with you on how to frozen the model to pb,please help me !
